### PR TITLE
paste: add page

### PR DESCRIPTION
**Pull request by chuanconggao**

### CODE REVIEW

File: pages/common/paste.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# paste
+
+> Merge lines of files.
+
+- join all the lines into one, using TAB as delimiter to separate the columns.
```
Comment (waldyrious) [inline pages/common/paste.md]: I think the "to separate the columns" is unnecessary and possibly even confusing. How about removing that part?

File: pages/common/paste.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`paste -s {{file}}`
+
+- join all the lines into one, using the specified delimiter to separate the columns.
```
Comment (waldyrious) [inline pages/common/paste.md]: same as above

File: pages/common/paste.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`paste -s -d {{delimiter}} {{file}}`
+
+- merge two files side by side, using TAB as delimiter to separate the columns.
```
Comment (waldyrious) [inline pages/common/paste.md]: here too

**Comment by chuanconggao** [comment]
@waldyrious Thanks for the suggestions. I have made some changes.

File: pages/common/paste.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- join all the lines into one, using TAB as delimiter.
+
+`paste -s {{file}}`
+
+- join all the lines into one, using the specified delimiter.
+
+`paste -s -d {{delimiter}} {{file}}`
+
+- merge two files side by side, using TAB as delimiter. Lines in the first file and the second file form the first column and the second column, respectively.
```
Comment (igorshubovych) [inline pages/common/paste.md]: Think about how to make the description shorter.

E.g.
- Merge two files side by side, each in its column, using TAB as delimiter.

Or propose your example

File: pages/common/paste.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+`paste {{file1 file2}}`
```
Comment (igorshubovych) [inline pages/common/paste.md]: Should be 
`paste {{file1}} {{file2}}`

File: pages/common/paste.md (hunk 4 above)
Comment (waldyrious) [inline pages/common/paste.md]: Brilliant conciseness skills, @igorshubovych!

**Comment by waldyrious** [comment]
It looks much better now :) One question though: does -s stand for "single line"? If so, the example description should use that expression, as it makes it easier to understand what the option does and to memorize its short name.

**Comment by chuanconggao** [comment]
@waldyrious In its manpage, `-s` actually stands for `--serial`. I think using that may cause some confusion and need more explanation.

**Comment by waldyrious** [comment]
Well, I think we can take the liberty of calling it "single line" as it is helpful for beginners, a good mnenomic, and technically it isn't incorrect. What do you think? "join all the lines into a single line, using TAB as delimiter."

**Comment by chuanconggao** [comment]
@waldyrious Sorry, I misunderstood your before. Your suggestion does sound more clear. I have made the change.

**Comment by waldyrious** [comment]
Ok, I'm very happy with how this page turned out, pretty simple and intuitive as all TLDRs should be! LGTM :+1:

**Comment by igorshubovych** [comment]
LGTM. But please, squash the commits.

**Comment by chuanconggao** [comment]
@igorshubovych It is my first time using rebase, and it took me quite a while. Now it is clean.

**Comment by felixonmars** [comment]
LGTM, thanks :)

**Comment by waldyrious** [comment]
@chuanconggao while it's fresh on your mind, take a look at the candidate `tldr git-rebase` (#473) in case it could be improved to make it easier for others :)

**Pull request merged**